app/controllers/admin/results_controller.rb

```diff
@@ -24,15 +24,17 @@ def create
 
     # GET /admin/results/search
     def search
-      search_string = search_params[:domain_name]
-      statuses_contains = params[:statuses_contains]
+      @_params = search_params.compact_blank
```

Comment:
If you do not plan to use class variables in the view, then it is better not to do so. Because, I expect (or someone else) that they are also used in view and logic is built on them, but in reality they are not there.